install.sh

```diff
@@ -105,7 +105,7 @@ case "${ARCH}" in
   # x86_64 and aarch64 userspace
   # shellcheck disable=SC2046
   [ $(od -An -t x1 -j 4 -N 1  /bin/bash) == 02 ] && LIB_SUFFIX='64'
-  if [[ $LIB_SUFFIX == '64' ]] && [[ $ARCH == 'aarch64' ]]; then
+  if [[ $LIB_SUFFIX == '64' ]] && { [[ $ARCH == 'armv7l' ]] || [[ $ARCH == 'armv8l' ]] || [[ $ARCH == 'aarch64' ]]; } ; then
```

Comment:
`{ [[ "$ARCH" =~ ^(armv7l|armv81|aarch64)$ ]] }; then`?

Reply:
What's the problem?
```
  if [[ $LIB_SUFFIX == '64' ]] && { [[ $ARCH == 'armv7l' ]] || [[ $ARCH == 'armv8l' ]] || [[ $ARCH == 'aarch64' ]]; } ; then
    echo_error "Your device is not supported by Chromebrew yet, installing as armv7l."
    LIB_SUFFIX=
    ARMV7LONAARCH64=1
  fi
```

Reply:
https://www.shellcheck.net/wiki/SC2235

Reply:
Semicolon location is intentional. As per https://www.shellcheck.net/wiki/SC2235 : 

_You appear to be using (..) to group test commands. This creates a subshell, making it unnecessarily slow. Avoid this by using { ..; } to group.

**Be careful to note that unlike (..), this requires both a space after the { and a semicolon before the }._**

Reply:
Seems to work!

Reply:
Made the change.